src/app/components/Resume/ResumePDF/ResumePDFProfile.tsx

```diff
@@ -52,18 +52,35 @@ export const ResumePDFProfile = ({
             }
           }
 
-          const shouldUseLinkWrapper = ["email", "url"].includes(key);
+          const shouldUseLinkWrapper = ["email", "url", "phone"].includes(key);
           const Wrapper = ({ children }: { children: React.ReactNode }) => {
             if (!shouldUseLinkWrapper) return <>{children}</>;
 
-            return (
-              <ResumePDFLink
-                src={key === "email" ? `mailto:${value}` : `https://${value}`}
-                isPDF={isPDF}
-              >
-                {children}
-              </ResumePDFLink>
-            );
+            switch (key) {
+              case "email":
+                return (
```

Comment:
nit: We might be able to remove some component redundancy if we move the component return to the end and let the switch statement focus on handling the `src`. What do you think?

```typescript
// Something like this
let src = "";
switch (key) {
  case "email": {
    src = `mailto:${value}`;
    break;
  }
  case "phone": {
    src = ``; 
    break;
  }
  default: {
    src = ``;
  }
}

return (
  <ResumePDFLink src={src} isPDF={isPDF}>
    {children}
  </ResumePDFLink>
);
```

Reply:
That's a great idea, I'll make a commit real quick.